src/admin-views/editor/button-view-orders.php

```diff
@@ -26,9 +26,12 @@
  */
 $url = apply_filters( 'tribe_filter_attendee_order_link', $url, $post->ID );
 ?>
-<a
-	href="<?php echo esc_url( $url ); ?>"
-	class="button-secondary"
->
-	<?php esc_html_e( 'View Orders', 'event-tickets' ); ?>
-</a>
+
+<?php if ( ! empty( $url ) ) : ?>
+	<a
+			href="<?php echo esc_url( $url ); ?>"
```

Comment:
this seems like a couple tabs too many to my eye, but *totally* not a blocker (and may be intentional)...

Reply:
indeed too much tabbing around.